.circleci/config.yml

```diff
@@ -315,6 +315,43 @@ jobs:
       #       cp ../src/js/*.js node_modules/pyodide/
       #       npx webpack
 
+  test-cmdline-runner:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: .
+
+      - run:
+          name: test
+          command: |
+            set -x
+            export PYODIDE_ROOT=$(pwd)
+            echo $PYODIDE_ROOT
+            cd ~
+            mkdir test
+            cd test
+            git clone https://github.com/python-attrs/attrs --depth 1 --branch 22.1.0
+            git clone https://github.com/zopefoundation/zope.interface.git --depth 1 --branch 5.4.0
+            git clone https://github.com/nedbat/coveragepy.git --depth 1 --branch coverage-5.5
```

Comment:
Why do we need these 3 ?

Reply:
Dependencies installed by `attrs[tests]`. Would be great to replace the build front end used by pip with `pyodide build` then it could do all this automatically. But it's not clear that's possible without unstable modifications to pip.

Reply:
Why do we need `attrs[tests]` though? Can we at least use `attrs[tests-no-zope]` without zope?